xbmc/utils/FileUtils.cpp

```diff
@@ -353,3 +353,20 @@ bool CFileUtils::Exists(const std::string& strFileName, bool bUseCache)
 {
   return CFile::Exists(strFileName, bUseCache);
 }
+
+
+bool CFileUtils::DeleteWithConfirm(const std::shared_ptr<CFileItem>& item)
+{
+  const bool confirm = CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
+      CSettings::SETTING_FILELISTS_CONFIRMFILEDELETE);
+
+  if (confirm)
```

Comment:
You could save the `confirm`variable and just write

```
if (CServiceBroker::GetSettingsComponent...)
```

Reply:
I know. I think it is more readable this way.